frontend/src/metabase/common/components/ConfirmModal/ConfirmModal.tsx

```diff
@@ -51,6 +53,7 @@ export const ConfirmModal = ({
       <Flex direction="column" gap="lg" mt="lg">
         {content ? <Text>{content}</Text> : null}
         <Text>{message}</Text>
+        {children}
```

Comment:
Do we need `content`, `message`, *and* `children` props here? Can we at least reduce it to 2 props?